core/src/display_object/avm2_button.rs

```diff
@@ -805,6 +805,7 @@ impl<'gc> TInteractiveObject<'gc> for Avm2Button<'gc> {
     }
 
     fn mouse_cursor(self, _context: &mut UpdateContext<'_, 'gc, '_>) -> MouseCursor {
+        // TODO: Should we also need to check for the `enabled` property like AVM1 buttons?
```

Comment:
The `buttonMode` property also applies in AVM2, but otherwise yep, if the `enabled` is false, then the hand cursor shouldn't appear.